sql/schema.sql

```diff
@@ -6,22 +6,26 @@
 -----------------
 
 CREATE TABLE IF NOT EXISTS repos
-( id         SERIAL PRIMARY KEY  
-, owner      TEXT   NOT NULL       
-, name       TEXT   NOT NULL
-, descr      TEXT
-, categories TEXT   ARRAY
+( id         SERIAL    PRIMARY KEY  
+, owner      TEXT      NOT NULL       
+, name       TEXT      NOT NULL
+, descr      TEXT      NOT NULL
+, categories TEXT      ARRAY
+, created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW()
```

Comment:
Yeah, in all cases using `TIMESTAMP WITH TIME ZONE` is a good default. But let's alos add not-nullable constraint to every timestamp column.
```suggestion
, created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW() NOT NULL
```